### DISCUSSION TITLE
Please update Nuget if possible

### DISCUSSION THREAD

**Discussion by RobertBeekman** [amwx/FluentAvalonia#342] [General]
Hello,

I've (perhaps unwisely) moved my application over to the preview builds and people have been getting some crashes with context menus. I noticed these are fixed in 1afc0b0676ced185ab70f36fc7f1e363240ca191 but this is as far as I can tell not included in preview6

So, if you could release a preview6.1 with the latest master fixes, that'be awesome :)

**Comment by timunie** [comment]
> So, if you could release a preview6.1 with the latest master fixes, that'be awesome :)

~That's unlikely. We have enough to do with getting things done for 11.0 final release. Making a new preview release takes up capacity which we don't want to spend. We could do it every 2 days if you ask all communtiy memebers 🤣 . Anyway, if you need a fix urgent I really suggest to use nightly builds instead of preview builds. See https://github.com/AvaloniaUI/Avalonia/wiki/Using-nightly-build-feed~

That was bullshit, sorry ^^

**Reply by RobertBeekman** [reply]
Hey, please note this is a request for FluentAvalonia, not Avalonia itself. If the same applies to that, that's of course fine :)

**Reply by timunie** [reply]
woah, didn't notice that we are in Fluent Avalonia rn. Sorry. 

Tbh, I think for FluentAvalonia I cloned the source for now and build it on my own to keep track. Sorry for the confusion.

**Comment by robloo** [comment]
~I agree it would be great to do another interim release; however, the time that takes is not insignificant and @amwx probably needs a break. There are still a lot of breaking changes going on upstream as well. Hopefully after RC things will slow down again.~

~I did take a look at upgrading the code and there are a lot of things broken again, some of them I'm not sure how to fix at a quick glance (like Window API changes). (Excluding ItemsControl changes) Most of the issues are with TopLevel / Window APIs that are getting locked down quite a bit. Needed functionality is going private and that always take time to work-around (or argue about adding back upstream).~

Edit: Nevermind, seems a PR is already open working on a lot of this: https://github.com/amwx/FluentAvalonia/pull/345 !

**Comment by RobertBeekman** [comment]
Thanks for your hard work @amwx!
